docker-for-windows/index.md

```diff
@@ -478,15 +478,15 @@ You can configure options on the Docker daemon in the given JSON configuration f
 
 ![Docker Daemon](images/docker-daemon.png)
 
-For a full list of options on the Docker daemon, see <a href="https://docs.docker.com/engine/reference/commandline/dockerd/" target="_blank">daemon</a> in the Docker Engine command line reference.
+For a full list of options on the Docker daemon, see [daemon](/engine/reference/commandline/dockerd/) in the Docker Engine command line reference.
 
 In that topic, see also:
 
-* [Daemon configuration file](https://docs.docker.com/engine/reference/commandline/dockerd/#/daemon-configuration-file)
+* [Daemon configuration file](/engine/reference/commandline/dockerd/#/daemon-configuration-file)
```

Comment:
@mstanleyjones well if I do this the redirect loses the anchor link. You can try by yourself:

https://docs.docker.com/engine/reference/commandline/dockerd/#/daemon-configuration-file

versus

https://docs.docker.com/engine/reference/commandline/dockerd.md#/daemon-configuration-file